src/js/utils/ErrorMessageUtil.js

```diff
@@ -17,6 +17,9 @@ const ErrorMessageUtil = {
       return rule.match.exec(pathString);
     });
 
+    if (error.unanchored) {
```

Comment:
I think some name here is really misleading, either it's `getUnanchoredErrorMessage` or `unanchored`.
When I read this method I understand that `getUnanchoredErrorMessage` and I wouldn't understand why I would need to add a `unanchored` flag on the error itself. If there is a good explaination I would be fine as long as we add it to the jsdoc string, e.g.

```js
/**
* @param {Object} error - The error message
* @param {Boolean} error. unanchored - <why the flag is needed>
*/
```

Reply:
I'm with you that this is a bit confusing, but there is a Google Document which explains how these errors work, I've added the "unanchored" property there and explained what it is. 
https://docs.google.com/document/d/1kwdPxBo3UlRZvMxLLIUqSZqLlUJitEuIAWderUgWY1c/edit#heading=h.pw1xuuy946c9

Reply:
Wow, that is super interesting. It makes me a bit sad that we can't put the link in the JSDoc because it would be super helpful. Any idea on how we might make this information more accessible?

Reply:
@orlandohohmeier mentioned a JIRA for refactoring this, we should keep this in mind.

Reply:
Shall we call it `isUnanchored`? It's kind of a convention in Javascript calling boolean flags starting with `is<*>`

Reply:
changed 👍